### ISSUE TITLE
Setting up Match Schedule with wrong values returns no error message

### ISSUE THREAD

**Issue by emmell620** [FIRST-Tech-Challenge/scorekeeper#486]
No error messages get displayed if there is a problem with creating the match schedule. It just sits there without generating the schedule leaving the user wondering if they hit the button or not.  An error message should be returned.    Steps to recreate: 1) 6 matches per team; 2) create first block 10:30-12:00; 3) Add Break 60 minutes starting at 12:00. 4) Add Match Block starting at 12:45 for the remainder of the matches.  5) Press Save.  Nothing happens.  The user has to look real hard to find the trouble that he created a 60 minute lunch break and started the matches 15 minutes earlier than normal.  A message that says "Conflicting times" or something like that would be helpful.  (Version 1.32).

**Comment by adwiii** [comment]
It appears that attempting to run MatchMaker when the schedule is an invalid state like you describe will give you a nondescript error saying "Error running MatchMaker. Please check settings and try again.", but as you note the save silently fails. We can work to 1) make the error more descriptive and try to provide information about where specifically the error is and 2) have the error message display for both saving and generating matches.

**Comment by adwiii** [comment]
This has been addressed in `v1.4.0`. The system now alerts you on save, preview, and MatchMaker run if there is an issue with the schedule, and gives you a more detailed explanation as to what is incorrect. In the image below, I have set the start time for the second block of matches to be `23:30` whereas the break ends at `23:45`. Doing so will give the error:
> Schedule out of order or overlapped: Qualification 16 starts at 2020/02/05 23:30 but the previous block ended at 2020/02/05 23:45

![image](https://user-images.githubusercontent.com/9453262/73899541-fac9b100-485a-11ea-8d4b-664761c9668e.png)
